### PR TITLE
Remove linking prompt

### PR DESCRIPTION
**Pull request by dtrabo**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Modified postlink.js files, deleted any prompts from them

### CODE REVIEW

File: appcenter-link-scripts/src/android/index.js [hunk 1]
```diff
@@ -18,7 +15,7 @@ module.exports = {
         return false;
     },
 
-    initAppCenterConfig() {
+    /* initAppCenterConfig() {
```
Comment (MatkovIvan) [inline appcenter-link-scripts/src/android/index.js]: Please delete it instead of commenting

File: appcenter-link-scripts/src/ios/index.js [hunk 2]
```diff
@@ -28,7 +26,7 @@ module.exports = {
         return true;
     },
 
-    initAppCenterConfig() {
+    /* initAppCenterConfig() {
```
Comment (annakocheshkova) [inline appcenter-link-scripts/src/ios/index.js]: Remove comments

File: appcenter-link-scripts/src/android/index.js [hunk 3]
```diff
@@ -18,28 +15,6 @@ module.exports = {
...
-            name: 'app_secret',
-        }]).then((answers) => {
-            config.set('app_secret', answers.app_secret);
-            const file = config.save();
-            console.log(`App Secret for Android written to ${file}`);
-            return file;
-        });
-    },
-
     patchStrings(key, value) {
```
Comment (annakocheshkova) [inline appcenter-link-scripts/src/android/index.js]: Is this used?

File: appcenter-analytics/scripts/postlink.js [hunk 4]
```diff
@@ -1,64 +1,19 @@
...
-        });
-} else {
-    promise = Promise.resolve();
+    rnpmlink.android.removeAndroidDuplicateLinks();
 }
+promise = Promise.resolve();
 
 // Then iOS even if Android failed.
 if (rnpmlink.ios.checkIfAppDelegateExists()) {
     promise
```
Comment (roland-chernov-akvelon) [inline appcenter-analytics/scripts/postlink.js]: I think we can replace it with just:
```suggestion
    rnpmlink.ios.initInAppDelegate('#import <AppCenterReactNativeAnalytics/AppCenterReactNativeAnalytics.h>', code, /.*\[AppCenterReactNativeAnalytics register.*/g).then // ...
```
And get rid of the `promise` variable (lines 9, 4, 34).

I'm not sure, but `return promise` on line 34 doesn't make sense. `module.exports = promise` should be used if we want to use this promise outside of this module.

File: appcenter-crashes/scripts/postlink.js [hunk 5]
```diff
@@ -1,64 +1,18 @@
...
-            rnpmlink.android.removeAndroidDuplicateLinks();
-        }).catch((e) => {
-            console.error(`Could not configure AppCenter Crashes for Android. Error Reason - ${e.message}`);
-            return Promise.resolve();
-        });
-} else {
-    promise = Promise.resolve();
+    rnpmlink.android.removeAndroidDuplicateLinks();
 }
+promise = Promise.resolve();
```
Comment (roland-chernov-akvelon) [inline appcenter-crashes/scripts/postlink.js]: The same here. Please see my comment above.

**Pull request closed without merging**